src/screens/MainApp/DiagnoseRequest/MainScreen/styles.js

```diff
@@ -0,0 +1,38 @@
+import { StyleSheet } from 'react-native'
+import { theme } from '~/constants'
+import { moderateScale } from 'react-native-size-matters'
+
+const styles = StyleSheet.create({
+  container: {
+    padding: theme.sizes.padding,
+    flex: 1,
+    flexDirection: 'column',
+    alignItems: 'center'
+  },
+  image: {
+    margin: theme.sizes.margin,
+    width: moderateScale(150),
+    height: moderateScale(150),
+    backgroundColor: theme.colors.gray,
+    borderRadius: moderateScale(10)
+  },
+  title: {
+    ...theme.fonts.h1,
```

Comment:
While this still works, I believe it may be redundant as you applied the fontVariant='h1' in the component.